tests/Fixtures/app/console

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env php
```

Comment:
That makes sense to put it here. If you put something in `bin` it looks like you are exposing it too much (in a way of implicitly proposing to use it) to me.